packages/cxl-ui/scss/global/cxl-marketing-nav.scss

```diff
@@ -62,3 +62,21 @@ vaadin-context-menu-item[theme~="cxl-marketing-nav"] {
     }
   }
 }
+
+.vaadin-context-menu-item--description {
+  margin-top: var(--lumo-space-s);
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-s);
+  color: var(--lumo-secondary-text-color);
+}
+
+// Restricts menu item description width to max of parent width.
+vaadin-context-menu-list-box {
+  --cxl-vaadin-context-menu-item: fit-content;
+}
+
+@media (min-width: 420px) and (min-height: 420px) {
```

Comment:
I'm not sure we should be inventing random breakpoints.

Only thing to react to should be `cxl-app-layout[wide]`.

Reply:
> I'm not sure we should be inventing random breakpoints.
> 
> Only thing to react to should be `cxl-app-layout[wide]`.

Not random, `wide` is not applicable here.

See from upstream:
<img width="300px" src="https://user-images.githubusercontent.com/1642796/189652229-72a6a8fc-57cd-4595-9cbf-abe8cd903fa9.png" />

Reply:
You should not be looking at legacy Lumo Styles repo.

What's the status at https://github.com/vaadin/web-components/tree/master/packages/vaadin-lumo-styles

Reply:
Looks same: https://github.com/vaadin/web-components/blob/master/packages/vaadin-lumo-styles/mixins/menu-overlay.js#L47

![chrome_epNxvkD4wE](https://user-images.githubusercontent.com/1642796/189657944-5941feb9-e373-4906-a327-905321a0acd4.png)